atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/LockWatchManagerImplTest.java

```diff
@@ -75,6 +77,9 @@ public void before() {
     @Test
     public void createMethodParsesTablesFromSchema() {
         when(schema.getLockWatches()).thenReturn(ImmutableSet.of(fromSchema));
+        when(fromSchema.accept(LockWatchReferencesVisitor.INSTANCE))
```

Comment:
necessary because the wiring change in `ValueStoreImpl` caused a NPE because everything is mocked in this test class.